journal/journal.go

```diff
@@ -74,7 +90,12 @@ func Enabled() bool {
 // for more details.  vars may be nil.
 func Send(message string, priority Priority, vars map[string]string) error {
 	if conn == nil {
-		return journalError("could not connect to journald socket")
+		return errors.New("could not initialize socket")
+	}
+
+	socketAddr, err := net.ResolveUnixAddr("unixgram", journalSocket)
```

Comment:
Is it needed to ResolveUnixAddr on every Send() call ? Since journalSocket is not going to change once the library is inited.

Reply:
Indeed, there is no need for `ResolveUnixAddr`, we can just create our `UnixAddr` struct manually here instead.